landing-page/content/common/roadmap.md

```diff
@@ -30,33 +30,18 @@ Related design docs will be linked on the planning boards.
 # Priority 1
 
 * API: [Iceberg 1.0.0](https://github.com/apache/iceberg/projects/3) [medium]
-* Spark: [Merge-on-read plans](https://github.com/apache/iceberg/projects/11) [large]
-* Maintenance: [Delete file compaction](https://github.com/apache/iceberg/projects/10) [medium]
-* Flink: [Upgrade to 1.13.2](https://github.com/apache/iceberg/projects/12) (document compatibility) [medium]
 * Python: [Pythonic refactor](https://github.com/apache/iceberg/projects/7) [medium]
+* Spec: [Z-ordering / Space-filling curves](https://github.com/apache/iceberg/projects/16) [medium]
+* Spec: [Snapshot tagging and branching](https://github.com/apache/iceberg/projects/4) [small]
+* Views: [Spec](https://github.com/apache/iceberg/projects/6) [medium]
```

Comment:
shall we also add the Puffin stats stuff here?

Reply:
I don't see a project for this in GitHub, should I just add it with a link to PR [#4741](https://github.com/apache/iceberg/pull/4741)?

Reply:
makes sense IMO

Reply:
Updated! 👍